### PR TITLE
Sync data files with hub index 

### PR DESCRIPTION
**Pull request by sbs2001**
See https://github.com/crowdsecurity/crowdsec/pull/1153

### CODE REVIEW

File: ci.go [hunk 1]
```diff
@@ -57,8 +64,13 @@ var types = []string{
 	"scenarios",
 	"postoverflows",
 	"collections",
+	"data_files",
 }
 
+var skipDataFiles bool
+
+// var dataFileConfigs = make([]dataFileConfig, 0)
```
Comment (AlteredCoder) [inline ci.go:72]: remove commented line please

File: ci.go [hunk 2]
```diff
@@ -89,10 +101,11 @@ func main() {
 	flag.StringVar(&outFile, "output", ".index.json", "File to output index")
 	flag.BoolVar(&generate, "generate", false, "File to output index")
 	flag.StringVar(&inputFile, "input", ".index.json", "File to read index from")
+	flag.BoolVar(&skipDataFiles, "no-data-files", false, "Don't Download data files")
```
Comment (AlteredCoder) [inline ci.go:104]: Shouldn't we have a target "data_file" like for bouncers or configuration file instead of this flag?

File: generate.go [hunk 3]
```diff
@@ -71,51 +81,62 @@ func (ti *typeInfo) generate(filepath string, configType string) (string, error)
...
+		if fInfo.Description != "" {
+			ti.Description = fInfo.Description
 		}
-		if len(fInfo.Scenarios) > 0 {
-			ti.Scenarios = fInfo.Scenarios
+
+		if fInfo.Labels != nil {
+			ti.Labels = fInfo.Labels
+
+			// var tags_to_keep = []string{"service", "type"}
```
Comment (AlteredCoder) [inline generate.go:117]: remove commented lines please

File: generate.go [hunk 4]
```diff
@@ -192,3 +221,71 @@ func generateIndex(configType string) (map[string]typeInfo, error) {
...
+
+func getDataFileNameFromPath(p string) string {
+	splits := strings.Split(p, "/")
+	n := len(splits)
+	author := splits[n-2]
+	name := strings.Split(splits[n-1], ".")[0]
+	return author + "/" + name
+}
+
+func fastlyDfProcessor(data []byte) ([]byte, error) {
```
Comment (AlteredCoder) [inline generate.go:269]: i think we want to have the generation/formating of list in a different binary/code that can be run without the index.json generation

File: generate.go (hunk 3 above)
Comment (sbs2001) [reply generate.go:117]: This was present before the PR, hence  didn't touch it. Are you sure we should remove this ?

File: .index.json [hunk 5]
```diff
@@ -45,25 +45,9 @@
   },
   "crowdsecurity/base-http-scenarios": {
    "path": "collections/crowdsecurity/base-http-scenarios.yaml",
-   "version": "0.5",
+   "version": "0.1",
```
Comment (AlteredCoder) [inline .index.json:48]: all the parsers/scenario have been reset to v0.1 ?

**Pull request merged**